web-api/terraform/bin/deploy-app.sh

```diff
@@ -114,5 +114,5 @@ export TF_VAR_bounced_email_recipient=$BOUNCED_EMAIL_RECIPIENT
 export TF_VAR_scanner_resource_uri=$SCANNER_RESOURCE_URI
 
 terraform init -backend=true -backend-config=bucket="${BUCKET}" -backend-config=key="${KEY}" -backend-config=dynamodb_table="${LOCK_TABLE}" -backend-config=region="${REGION}"
-terraform plan
-terraform apply --auto-approve
+terraform plan -out execution-plan
+terraform apply --auto-approve execution-plan
```

Comment:
This was the fix to Terraform hanging — refreshing state twice in rapid succession caused Terraform to timeout in CircleCI (10 minutes without any command output). I hypothesize that the AWS API rate-limited Terraform, causing it to slow down, but I never 100% figured it out. This reliably works though, so I went with it!